.github/workflows/pr.workflow.yml

```diff
@@ -70,5 +70,6 @@ jobs:
           find .
           export PYTHONPATH=`echo dione-spark/target/dione-spark-*-SNAPSHOT.jar`
           python -V
+          pip install pypandoc==1.7.5
```

Comment:
where is this used at?

Reply:
In the GitHub CI - the package is installed before tests are run. it's necessary for them to succeed.